app/models/user.rb

```diff
@@ -422,6 +426,16 @@ def va_profile_v2_email
     vaprofile_contact_info&.email&.email_address
   end
 
+  def va_profile_phone
+    home = vaprofile_contact_info&.home_phone
+    home.area_code + home.phone_number
```

Comment:
Is there a reason this function isn't doing the same things as `va_profile_mobile_phone`? (Doing `.to_s` on the area code/ phone number)